x-pack/plugins/cases/public/components/connector_selector/form.tsx

```diff
@@ -53,7 +51,7 @@ export const ConnectorSelector = ({
     [handleChange, field]
   );
 
-  return isEdit ? (
```

Comment:
There is no need for this component to do this check. Either the consumers should do it (they do in this RP) or it can be part of a bigger component that does this check.